test/compilable/test16607.d

```diff
@@ -0,0 +1,15 @@
+struct A(T)
+{
+    T t; // causes A to be SIZEOKfwd b/c B (passed as T) isn't yet done
```

Comment:
Thank you! But would it be much to ask to avoid shortening words in commits in the future? They might not be very accessible to everyone depending on their English level.